doc/btp_spec.txt

```diff
@@ -1903,3 +1903,997 @@ Events:
 		Retry parameter is a binary value indicaing that a poll was a retry.
 
 		This event indicates that the IUT as an LPN has polled a Friend.
+
+Mesh Model Service (ID 5)
+==================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Generic OnOff State Get/Status
+
+		Controller Index:		<controller id>
+	    	Command parameters: 		<none>
+		Response Status parameters:	Present OnOff (1 octet)
+		                           	Target OnOff (1 octet)
+		                            	Remaining Time (4 octets)
+            	Valid OnOff values: 		0x00 = Off
+				        	0x01 = On
+
+        	This command is used to get the OnOff state.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x03 - Generic OnOff State Set/Status
+
+	    	Controller Index:		<controller id>
+		Command parameters: 		Acknowledgement (1 octet)
+		                    		OnOff (1 octet)
+		                    		Transition Time (1 octet)
+		                    		Delay (1 octet)
+		Response Status parameters: 	Present OnOff (1 octet)
+		                            	Target OnOff (1 octet)
+		                            	Remaining Time (4 octets)
+        	Valid OnOff values: 		0x00 = Off
+					        0x01 = On
+        	Acknowledgment values: 		True = with requesting a status response
+                               			False = without requesting a status response
```

Comment:
indent. I have a feeling it would be better to stick to decimal definitions like 0x00 and 0x01 instead